src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java

```diff
@@ -29,8 +29,19 @@ public static UploadResponse unsuccessfulDocumentManagementUploadResponse() {
     }
 
     public static StartEventResponse successfulCoreCaseDataStoreStartResponse() {
-        String response = new ResourceReader().read("/core-case-data/start-response.success.json");
-        return jsonMapper.fromJson(response, StartEventResponse.class);
+        return jsonMapper.fromJson(startEventResponseSuccessAsFile(), StartEventResponse.class);
+    }
+
+    public static String startEventResponseSuccessAsFile() {
```

Comment:
After introducing the builders, do we need these `JSON` resources?

Reply:
removed